modules/oe_translation_poetry/src/Form/PoetryCheckoutFormBase.php

```diff
@@ -341,7 +346,8 @@ protected function cancelAndRedirect(FormStateInterface $form_state): void {
       $job->delete();
     }
 
-    $this->queue->reset();
+    $queue = $this->queueFactory->get($form_state->get('entity'));
```

Comment:
what?

Reply:
In this method in [line 344](https://github.com/openeuropa/oe_translation/blob/c50fceee198be5ef4e218a5cf19aa9591f85cd95/modules/oe_translation_poetry/src/Form/PoetryCheckoutFormBase.php#L342) we still have:
```$jobs = $this->queue->getAllJobs();```

Now there is no property `$this->queue` .

So we must have the line:
```
$queue = $this->queueFactory->get($form_state->get('entity'));
```
... in the beginning of the method and use `$queue` instead of `$this->queue`.

Reply:
aaa yes, @imanoleguskiza please fix that. That will crash.